style.css

```diff
@@ -0,0 +1,330 @@
+font-weight: 500;
+font-size: 16px;
+line-height: 24px;
+text-align: center;
+color: #595959;
+margin-left: -500px;
+}
+.listcard {
+margin-left: 430px;
+width: 335px;
+height: 257px;
+}
+.card {
+border: solid #fdfafa;
+background-color: #FFFFFF;
+margin-top: 10px;
+}
+.listtop {
+width: 335px;
+height: 4px;
+background-color: #25A59E;
+}
+.title {
+font-family: "Gilda display";
+font-weight: 400;
+font-size: 28px;
+line-height: 50px;
+width: 295px;
+color: #262626;
+}
+.description {
+width: 295px;
+font-family: "urbanist";
+font-weight: 400;
+font-size: 15px;
+line-height: 23px;;
+}
+.text {
+font-family: "Gilda display";
+font-weight: 400;
+font-size: 15px;
+line-height: 23px;
+width: 84px;
+color: #25A59E;
+margin-left: 170px;
+}
+.forward {
+margin-left: 270px;
+transform: translateY(-20px);
+width: 16px;
+height: 16px;
+}
+.card-1 {
+  background-color: #25A59E;
+}
+.title-1 {
+font-family: "Gilda display";
+font-weight: 400;
+font-size: 28px;
+line-height: 40px;
+color: #FFFFFF;
+}
+.description-1 {
+font-family: "Urbanist";
+font-weight: 400;
+font-size: 16px;
+line-height: 24px;
+width: 295px;
+color: #FFFFFF;
+}
+.text-1 {
+font-family: "Gilda display";
+font-weight: 400;
+font-size: 15px;
+line-height: 17.66px;
+margin-left: 150px;
+color: #FFFFFF;
+}
+.memory {
+margin-top: 500px;;
+margin-left: 400px;
+}
+.make {
+font-family: "gilda display";
+font-weight: 400;
+font-size: 40px;
+line-height:  52px;
+}
+.thousands {
+font-family: "urbanist";
+font-weight: 500;
+font-size: 16px;
+line-height: 24px;
+margin-top: -10px;
+}
+.arrow {
+width: 335px; 
+height: 350px;
+color: #25A59E;
+}
+.travellers {
+width: 335px;
+margin-left: -350px;
+transform: translateY(-100px);
+}
+.trip {
+margin-left: 450px;
+font-family: "gilda display";
+font-weight: 400;
+font-size: 28px;
+line-height: 80px;
+margin-top: -80px;
+color: #262626;
+width: 335px;
+}
+.center {
+margin-left: 140px;
+}
+.movie {
+width: 375px;
+margin-left: 400px;
+}
+.header-container {
+margin-left: 400px;
+}
+.offer {
+font-family: "Gilda display";
+font-weight: 400;
+font-size: 40px;
+line-height: 52px;
+width: 335px;
+}
+.click {
+width: 147px;
+font-family: "urbanist";
+font-weight: 600;
+font-size: 18px;
+line-height: 50px;
+color: #FFFFFF;
+background-color:  #25A59E;
+margin-top: -450px;
+top: 0;
+left: 0;
+border: 0px;
+}
+.notification {
+margin-left: 400px;
+width: 375px;
+margin-top: 80px;
+}
+.movement {
+font-family: "Gilda display";
+font-weight: 400;
+font-size: 16px;
+line-height: 18.83px;
+}
+.agent {
+font-family: "urbanist";
+font-weight: 500;
+font-size: 16px;
+line-height: 24px;
+color: #595959;
+}
+.social-link {
+margin-left: 550px;
+}
+.bird {
+margin-top: -40px;
+margin-left: 50px;
+}
+.youtube {
+margin-top: -40px;
+margin-left: 70px;
+}
+.header-links {
+margin-left: 400px;
+display: inline-flex;
+gap: 30px;
+font-family: "urbanist";
+font-weight: 600;
+font-size: 16px;
+line-height: 24px;
+color: #595959;
+padding-top: 30px;
+}
```

Comment:
Hi @musyelevateher,

Good job so far!
You still need to work on some issues to get your project approved, but you are almost there!

<div style="text-align:center;">
    <img src="https://camo.githubusercontent.com/7cf1ba8db9cd08f454a9651e48128c064a4440a59881658c8593a41a8de243f3/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f744b797263304e546c4563475764546870342f67697068792e676966" alt="Remote Collaboration" style="width:40%;">
</div>

### To Highlight 💯
- **Clean code** 👍
- **Good use of Gitflow** 👍
- **Implemented some project requirements** 👍

### Required Changes ♻️
- Make your HTML truly semantic, ensure that the HTML tags correctly describe the content's meaning and structure.
- Please use professional PR title
- Make your design UI like the Figma design

# Figma design

![image](https://github.com/user-attachments/assets/2fdf7960-6e56-471c-98b9-d8397368488e)


![image](https://github.com/user-attachments/assets/7e9b1f95-b7c8-47f5-86c1-4cd7d33b2b3c)

# Your design


![image](https://github.com/user-attachments/assets/2b22c86d-09dc-41e6-92e8-433f4a610312)


![image](https://github.com/user-attachments/assets/f85b03ef-1145-431b-8f2f-6fb5dbf47d60)




Cheers and Happy coding! 👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.  
Please, remember to tag me in your question so I can receive the notification._